### PR TITLE
Add git difftool support (keybinding <c-t>)

### PR DESCRIPTION
**Pull request by zeevro**
I've added the ability to launch `git difftool` for what's currently in the diff screen.

### CODE REVIEW

File: pkg/gui/diffing.go [hunk 1]
```diff
@@ -159,3 +160,35 @@ func (gui *Gui) handleCreateDiffingMenuPanel() error {
 
 	return gui.c.Menu(types.CreateMenuOptions{Title: gui.c.Tr.DiffingMenuTitle, Items: menuItems})
 }
+
+func (gui *Gui) handleLaunchDiffTool() error {
+	gui.c.LogAction("Launch DiffTool")
+
+	cmdStr := "git difftool --submodule "
+
+	file := gui.currentlySelectedFilename()
```
Comment (Ryooooooga) [inline pkg/gui/diffing.go:169]:
```suggestion
	file := gui.currentlySelectedFilename()
	if file == "" {
		file = gui.getSelectedPath()
	}

```

for Staged/Unstaged Changes panel

File: pkg/gui/diffing.go [hunk 2]
```diff
@@ -159,3 +160,35 @@ func (gui *Gui) handleCreateDiffingMenuPanel() error {
...
+		cmdStr += gui.diffStr()
+	} else {
+		diffTerm := gui.currentDiffTerminal()
+		if diffTerm != "" {
+			cmdStr += fmt.Sprintf(" %s^ %s", diffTerm, diffTerm)
+		}
+	}
+
+	if file != "" {
+		cmdStr += " -- " + file
```
Comment (Ryooooooga) [inline pkg/gui/diffing.go:186]:
```suggestion
		cmdStr += " -- " + gui.os.Quote(file)
```

File: pkg/i18n/english.go [hunk 3]
```diff
@@ -953,6 +954,7 @@ func EnglishTranslationSet() TranslationSet {
 		DiffingMenuTitle:                    "Diffing",
 		LcSwapDiff:                          "reverse diff direction",
 		LcOpenDiffingMenu:                   "open diff menu",
+		LcLaunchDiffTool:                    "Launch git difftool",
```
Comment (mark2185) [inline pkg/i18n/english.go:957]: `Lc` in `LcSomeTitle` actually denotes `lowercase`, could you change it to `launch git difftool` then?

**Comment by barreiroleo** [comment]
Hello there! Any updates with this PR?

**Review by jesseduffield — CHANGES_REQUESTED** [review]
Marking as changes requested for my own bookkeeping. Let me know if you need any guidance with this @zeevro

**Comment by jesseduffield** [comment]
This is superceded by https://github.com/jesseduffield/lazygit/pull/3156. Closing

**Pull request closed without merging**